prisma/schema/application.prisma

```diff
@@ -22,6 +32,7 @@ model Application {
   tmuaPaper2Score       Decimal         @db.Decimal(2, 1)
   tmuaOverallScore      Decimal         @db.Decimal(2, 1)
   qualifications        Qualification[]
+  nextAction    NextAction        @default(TMUA_CANDIDATE)
```

Comment:
(silly one but) how about formatting this to align with the rest of the model? The autoformatter in WebStorm does the job for me.

Reply:
Resolved in [27caae9](https://github.com/imperial/ug-admissions/pull/3/commits/27caae9baf292fe5b258d24698129ed62193c569)